test/TestUtilities/SystemUtil.hpp

```diff
@@ -41,45 +41,23 @@ namespace sf
     }
 }
 
-// Utilities for approximate equality
-struct ApproxVec2
+template <typename T>
+struct Approx
 {
-    ApproxVec2(float x, float y)
-        : vector(x, y) {}
-
-    explicit ApproxVec2(const sf::Vector2f& v)
-        : vector(v) {}
-
-    sf::Vector2f vector;
+    Approx(const T& t) : value(t) {}
```

Comment:
Should this be explicit, or do we have places where we don't use the `Approx(expectedValue)` syntax?

Reply:
Yeah you're probably right. I can't think of a good reason you'd want to except an argument which wasn't precise the same same type as `T`.